arch/arm/src/cxd56xx/cxd56_cpu1signal.c

```diff
@@ -194,21 +192,18 @@ int cxd56_cpu1siginit(uint8_t sigtype, FAR void *data)
       goto err0;
     }
 
-  pthread_attr_init(&tattr);
-  tattr.stacksize      = CONFIG_CXD56CPU1_WORKER_STACKSIZE;
-  param.sched_priority = CONFIG_CXD56CPU1_WORKER_THREAD_PRIORITY;
-  pthread_attr_setschedparam(&tattr, &param);
+  pid = task_create("gnss_receiver", CONFIG_CXD56CPU1_WORKER_THREAD_PRIORITY,
```

Comment:
task_create is for userspace application, please replace with kthread_create instead.

Reply:
Dear @xiaoxiang781216 san,

Thank you very much for your suggestion.
I updated a change right now.

Could you check again?